.gitignore

```diff
@@ -9,3 +9,5 @@ node_modules
 vite.config.js.timestamp-*
 vite.config.ts.timestamp-*
 stats.html
+package-lock.json
+pnpm-lock.yaml
```

Comment:
These files are still being tracked. I think you've to clear git cache then commit again.

Reply:
Why is this in here?

This will break deoendabot

Reply:
removed in new commit